app/containers/Applications/ApplicationReviewContainer.tsx

```diff
@@ -0,0 +1,266 @@
+import React, {useState, SyntheticEvent} from 'react';
+import {Button, Modal, Dropdown, Form, Row, Col} from 'react-bootstrap';
+import {createFragmentContainer, graphql, RelayProp} from 'react-relay';
+import {
+  ApplicationReviewContainer_formResultStatus,
+  CiipFormResultStatus
+} from 'ApplicationReviewContainer_formResultStatus.graphql';
+import DropdownMenuItemComponent from 'components/DropdownMenuItemComponent';
+import {ReviewCommentType} from 'ApplicationCommentsByForm_reviewComment.graphql';
+import createReviewCommentMutation from 'mutations/application/createReviewCommentMutation';
+import createFormResultStatusMutation from '../../mutations/application/createFormResultStatusMutation';
+
+interface Props {
+  relay: RelayProp;
+  formResultStatus: ApplicationReviewContainer_formResultStatus;
+  formName: string;
+  formResultId: string;
+}
+
+interface Target extends EventTarget {
+  reviewComment: {
+    value: string;
+  };
+  reviewStatus: {
+    value: string;
+  };
+}
+
+export const ApplicationReview: React.FunctionComponent<Props> = ({
+  relay,
+  formResultStatus,
+  formName,
+  formResultId
+}) => {
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  const statusBadgeColor: Record<
+    CiipFormResultStatus,
+    'info' | 'danger' | 'success' | 'warning' | 'primary' | 'secondary'
+  > = {
+    IN_REVIEW: 'info',
+    NEEDS_ATTENTION: 'warning',
+    APPROVED: 'success',
+    CHANGES_REQUESTED: 'secondary'
+  };
+
+  const reviewStatuses: Record<string, CiipFormResultStatus> = {
+    approved: 'APPROVED',
+    requestChanges: 'CHANGES_REQUESTED',
+    inReview: 'IN_REVIEW',
+    needsAttention: 'NEEDS_ATTENTION'
+  };
+
+  const commentType: Record<string, ReviewCommentType> = {
+    APPROVED: 'APPROVAL',
+    IN_REVIEW: 'GENERAL',
+    CHANGES_REQUESTED: 'REQUESTED_CHANGE',
+    NEEDS_ATTENTION: 'INTERNAL'
+  };
+
+  const addComment = async (e: SyntheticEvent) => {
+    e.preventDefault();
+    e.persist();
+    const comment = (e.target as Target).reviewComment.value;
+    const status = (e.target as Target).reviewStatus.value;
+
+    if (!comment) return null;
+    const {environment} = relay;
+    const variables = {
+      input: {
+        applicationIdInput: formResultStatus.applicationId,
+        formIdInput: formResultStatus.formId,
+        descriptionInput: comment,
+        commentTypeInput: commentType[status],
+        versionNumberInput: formResultStatus.versionNumber
+      },
+      applicationId:
+        formResultStatus.formResultByApplicationIdAndVersionNumberAndFormId
+          .applicationByApplicationId.id,
+      version: formResultStatus.versionNumber.toString()
+    };
+
+    const response = await createReviewCommentMutation(
+      environment,
+      variables,
+      formResultId
+    );
+    console.log(response);
```

Comment:
handleClose is purposely not called here. In case a user wants to create several comments, or manually fix the status after a comment has changed it. A UX discussion could be had here, but the non-call is currently intentional